examples/networkpolicy/README.md

```diff
@@ -31,3 +31,32 @@ To understand how to deploy this template, please read the baseline [Kubernetes]
 Once the template has been successfully deployed, following the [simple policy tutorial](https://docs.projectcalico.org/v3.0/getting-started/kubernetes/tutorials/simple-policy) or the [advanced policy tutorial](https://docs.projectcalico.org/v3.0/getting-started/kubernetes/tutorials/advanced-policy) will help to understand calico networking.
 
 > Note: `ping` (ICMP) traffic is blocked on the cluster by default.  Wherever `ping` is used in any tutorial substitute testing access with something like `wget -q --timeout=5 google.com -O -` instead.
+
+### Update guidance for clusters deployed by acs-engine releases prior to 0.17.0
+Clusters deployed with calico networkPolicy enabled prior to `0.17.0` had calico `2.6.3` deployed, and a daemonset with an `updateStrategy` of `Ondelete`.
+
+acs-engine releases starting with 0.17.0 now produce an addon manifest for calico in `/etc/kubernetes/addons/calico-daemonset.yaml` contaning calico 3.1.x, and an `updateStrategy` of `RollingUpdate`.
+
+To get up and running with the new version of calico after upgrading a cluster with acs-engine `0.17.0` and up, follow these steps:
```

Comment:
Thank you @oivindoh! Looks great. Could you please add something like "As per the Calico v0.3.1 release notes, Calico v0.3.x includes breaking changes:
> Some highlights include:
> You must upgrade to Calico v2.6.5 before you can upgrade to v3.0.1 (see https://docs.projectcalico.org/v3.0/getting-started/kubernetes/upgrade/) 
> Calico deployments that access the etcd datastore directly must complete
a one-time migration.
> You must convert any customized Calico manifests via calicoctl convert
before you can use them with v3.0.1.

Here some instructions to get up and running with the new version of Calico […]" here?

Reply:
So people can have context on the breaking changes (and know it's a calico breaking change, not acs-engine)

Reply:
Good point! I added some text to be more explicit about this being due to calico breaking changes and not acs-engine 👍